app/src/main/java/com/teamagam/gimelgimel/app/network/services/IImageSender.java

```diff
@@ -0,0 +1,13 @@
+package com.teamagam.gimelgimel.app.network.services;
+
+import android.net.Uri;
+
+import com.teamagam.gimelgimel.app.view.viewer.data.geometries.PointGeometry;
+
+/**
+ * Created on 5/22/2016.
+ * Interface for sending and uploading images
+ */
+public interface IImageSender{
+    void sendImage(Uri imageUri, PointGeometry location, long imageTime);
```

Comment:
Is this interface for uploading an image or sending an ImageMessage?

Either way, renaming and documentation is needed !fix